packages/driver-interface/services/cache.js

```diff
@@ -1,22 +1,16 @@
 const bookingsCache = new Map()
 const vehiclesCache = new Map()
 const instructionsCache = new Map()
+const idMap = new Map()
```

Comment:
idMap should be renamed to something more explainable now that it has been moved to cache-file.

Reply:
Yes. My miss!